packages/web/examples/DataSearch/src/index.js

```diff
@@ -1,79 +1,95 @@
-import React, { Component } from 'react';
+import React from 'react';
 import ReactDOM from 'react-dom';
 
-import { ReactiveBase, DataSearch, ResultList, SelectedFilters } from '@appbaseio/reactivesearch';
+import {
+	ReactiveBase,
+	DataSearch,
+	ReactiveList,
+	ResultCard,
+	SelectedFilters,
+} from '@appbaseio/reactivesearch';
 
 import './index.css';
 
-class Main extends Component {
-	render() {
-		return (
-			<ReactiveBase
-				app="good-books-ds"
-				credentials="nY6NNTZZ6:27b76b9f-18ea-456c-bc5e-3a5263ebc63d"
-			>
-				<div className="row">
-					<div className="col">
-						<DataSearch
-							title="DataSearch"
-							dataField={['original_title', 'original_title.search']}
-							categoryField="authors.raw"
-							componentId="BookSensor"
-							URLParams
-						/>
-					</div>
+const Main = () => (
+	<ReactiveBase app="good-books-ds" credentials="nY6NNTZZ6:27b76b9f-18ea-456c-bc5e-3a5263ebc63d">
+		<div className="row">
+			<div className="col">
+				<DataSearch
+					title="DataSearch"
+					dataField={['original_title', 'original_title.search']}
+					categoryField="authors.raw"
+					componentId="BookSensor"
+					URLParams
+				/>
+			</div>
 
-					<div className="col">
-						<SelectedFilters />
-						<ResultList
-							componentId="SearchResult"
-							dataField="original_title"
-							size={10}
-							renderItem={this.renderData}
-							className="result-list-container"
-							pagination
-							react={{
-								and: 'BookSensor',
-							}}
-						/>
-					</div>
-				</div>
-			</ReactiveBase>
-		);
-	}
+			<div className="col">
+				<SelectedFilters />
+				<ReactiveList
+					componentId="SearchResult"
+					dataField="original_title"
+					size={10}
+					className="result-list-container"
+					pagination
+					react={{
+						and: 'BookSensor',
+					}}
+					render={({ data }) => (
+						<ReactiveList.ResultCardsWrapper>
```

Comment:
Shouldn't it be `ResultCardWrapper`?

Reply:
It's ok, we are using `ResultCardsWrapper`.